python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -191,6 +192,20 @@ def measure(self, measRecord, exposure):
         x1 = xc + dydtheta
         y1 = yc + dxdtheta
 
+        if not (0 <= x0 <= exposure.getBBox().endX and 0 <= x1 <= exposure.getBBox().endX
+                and 0 <= y0 <= exposure.getBBox().endY and 0 <= y1 <= exposure.getBBox().endY):
```

Comment:
Are you certain the edge points are appropriately 0 and not, e.g., `exposure.getBBox().startX` ?

Reply:
OIh, you are right, thanks for the catch. I swapped it this way as a test and missed swapping it back. I'll make sure to run Jenkins again to verify it doesn't change anything.